gapis/service/service.proto

```diff
@@ -1114,10 +1119,14 @@ message ProfilingData {
       AggregationOperator op = 5;
       string description = 6;
       bool select_by_default = 7;
-      // GPU counter group type specified by vendors.
-      repeated device.GpuCounterDescriptor.GpuCounterGroup counter_groups = 8;
```

Comment:
Maybe add
`reserved 8;`
?

Reply:
done.

Reply:
In this case it actually does not matter because we distribute server and client at the same time. We do not face a compatibility issue due to this.